Makefile

```diff
@@ -3,7 +3,7 @@
 PACK := github
 ORG := pulumi
 PROJECT := github.com/$(ORG)/pulumi-$(PACK)
-PROVIDER_PATH := provider/v4
```

Comment:
Have you relected this change in ci-mgmt yet?

Reply:
No, I forgot that was a thing. I'll check on that!

Reply:
https://github.com/pulumi/ci-mgmt/pull/260